src/walletConnect/screens/SessionRequest.tsx

```diff
@@ -21,8 +23,24 @@ function SessionRequest({ pendingSession }: Props) {
 
   const phoneNumber = useSelector(e164NumberSelector)
   const address = useSelector(currentAccountSelector)
+  const walletConnectDisplayedInfo = useSelector(walletConnectDisplayedInfoSelector)
   const isDappListed = useIsDappListed(name)
 
+  const requestDetails =
+    walletConnectDisplayedInfo === WalletConnectDisplayedInfo.None
+      ? []
+      : [
+          {
+            label: t('phoneNumber'),
+            value: phoneNumber,
+          },
+          {
+            label: t('address'),
+            value: address,
+            tapToCopy: true,
+          },
+        ]
```

Comment:
This will trigger later on when any value is used. 
I wonder if we should be more careful about backward compat for this and handle only cases we know about.

I'm anticipating this case: we add new enum value and don't realize it will display something unexpected for older clients.